generators/app/tasks/appSetup/coreFiles/packageJsonScripts.js

```diff
@@ -23,9 +23,13 @@ module.exports = function packgeJsonScripts() {
   packageJson.scripts['lint-diff'] =
     'git diff --staged --name-only --relative --diff-filter=ACM | grep -E "\\.(ts|tsx|js|jsx)$" | xargs eslint';
   packageJson.scripts['check-types'] = 'tsc';
+  packageJson.scripts['test:generate-coverage-file'] = 'node ./node_modules/@wolox/js-test-coverage-script';
+  packageJson.scripts['test:coverage'] =
+    'yarn run test --env=jsdom --coverage --passWithNoTests --watchAll=false --silent --coverageThreshold "{}" --coverageReporters="json-summary" && yarn run test:generate-coverage-file';
```

Comment:
1) Delete `packageJson.scripts.coverage = 'jest --coverage --passWithNoTests';` in line 8
2) Instead of `yarn run test` use directly: `'jest --passWithNoTests --env=jsdom --coverage --passWithNoTests --watchAll=false --silent --coverageThreshold "{}" --coverageReporters="json-summary" && yarn run test:generate-coverage-file'`

Reply:
done!